### DISCUSSION TITLE
Welcome to pyMECSim Discussions!

### DISCUSSION THREAD

**Discussion by kiranvad** [kiranvad/pyMECSim#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by karankumaryada** [comment]
MECSim file is a python file or what? why does it not have .py extension? It's not importing this correctly, hence not able to simulate.

**Comment by Helga-Y** [comment]
I tried to run an example. There is an error which raises on the moment sim.solve() as absence of log.txt file
